.github/workflows/continuous-integration.yml

```diff
@@ -9,18 +9,13 @@ concurrency:
   cancel-in-progress: true
 jobs:
   build_gradle:
-    name: "JDK ${{ matrix.java }} on ${{ matrix.os }}"
+    name: "${{ matrix.os }}"
```

Comment:
Can we actually keep the JDK version in this matrix?  I dream of running tests on JDK 17 in the not-too-distant future, at which point we'd just have to add this all back

Reply:
OK.